core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/db2/Db2DatabaseConnectionService.java

```diff
@@ -28,4 +29,8 @@ public String getDriver(Db2DatabaseConnection databaseConnection) {
         return Db2DatabaseConnection.class;
     }
 
+    @Override
+    public String refactorLimitAndOffset(Db2DatabaseConnection databaseConnection, String query) {
+        return query;
+    }
```

Comment:
All XDatabaseConnectionService's extend either DatabaseConnectionService or SchemaDatabaseConnectionService (which extends DatabaseConnectionService). You could implement the default behavior in the DatabaseConnectionService  class